packages/admin-ui-docs/src/components/TokensTable/tokens.tsx

```diff
@@ -77,6 +79,19 @@ export const shadow = shadowTokens.map(
   createMap('boxShadow', 'shadow', cssWithColorFormatter)
 )
 
+export const spacing = spaceTokens.map(
```

Comment:
What do you think about adding a visual example to the `Example` column? Something like this where the width represents the size space:
<img width="135" alt="Captura de Tela 2022-11-14 às 10 43 26" src="https://user-images.githubusercontent.com/20579226/201675285-654e20aa-320a-47ce-b8a5-5495487353c8.png">